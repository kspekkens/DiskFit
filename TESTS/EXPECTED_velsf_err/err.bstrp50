EXAMPLE/VELS/FITS/velsf_err.inp                                                                     
    lpa:  T    leps: T lcentre: T lsystemi T lradial: F lnax:    T luncert: T
#nunc   npars    ntot   nellip  nminr  nmaxr seed
      5      5     64     25      1     19    -50
 
 Bootstrap number:            0
   2.3561944961547852       0.44999998807907104        130.50000000000000        126.50000000000000       0.78539818525314331     
   69.6522217       132.914963       158.375046       169.940781       171.733856       181.612076       188.523697       183.723495       189.775818       192.026794       190.001724       192.188538       189.009201       189.795700       194.175552       194.324936       190.111633       193.582962       192.642288       192.469711       192.745712       193.448685       193.085007       194.532043       194.576248      -8.60850620      -24.3442497      -35.8591003      -42.1230965      -33.2658920      -48.0078659      -46.8348732      -43.2883301      -47.3781662      -43.0334282      -31.7133312      -20.2065983      -9.74966335      -2.92708492      -4.37177277      -3.53355694      0.135024518      -3.22860527     -0.209490821      -7.40719938      -18.7561378      -31.2330666      -32.0017395      -37.0687027      -42.3272095      -47.2677155      -40.8787537      -41.8959351      -38.4331207      -24.7098465      -22.4327278      -10.5811558      -7.75154161      -6.98168898      -5.74163246       1.15559947     -0.203367561     -0.766783237       100.007896    

 Bootstrap number:     1/    5  Chisq      0.90091
   2.2691751757190226       0.50350205860412345        128.54916331934817        128.59749918313668       0.41276003962337082     
   69.0246277       126.782372       153.435196       164.152756       163.060928       168.660828       175.715942       175.248047       177.425034       183.210190       181.375244       186.998001       186.309402       188.090927       193.812607       192.720230       190.551620       194.581787       191.013885       192.712555       191.581390       193.727325       192.948242       195.376846       193.788528       7.75784016       19.4287300       35.4761620       34.9939346       29.8163414       37.6222954       38.1249084       34.7952499       37.3398895       36.5069771       27.5713043       15.1968489       7.51237965       1.85154080       1.13507438       2.39424253       2.65480185       2.73404241     -0.847315609       4.94720554       12.4871702       27.0760269       25.0270901       30.2176018       32.1271400       38.5530815       36.7192993       31.3364601       30.2067242       19.5441322       18.9938488       7.03997421       9.56017113       6.91965389       2.14038825      0.309043705      0.998345971      -1.34751630       100.139450    

 Bootstrap number:     2/    5  Chisq      0.93312
   2.2698287617840620       0.50313539932838436        128.49268583573522        128.57917638975721       0.28630918507044290     
   71.0449295       136.135391       161.300552       171.873566       177.946442       189.075714       195.282043       190.660660       193.093246       199.432510       189.847321       203.156021       186.404083       192.437408       195.106308       195.093826       189.951187       194.313126       189.793289       193.379318       192.880020       194.366760       192.833710       194.490967       194.381378       9.47239590       25.2956848       39.1109810       42.9176064       39.4762878       54.0587959       50.6123543       51.6165123       50.7264519       46.3208733       34.7191200       26.4310760       7.82803345       6.99986410       5.75469828       4.70901966       1.02157640       1.66086650      0.313815713       7.33430147       21.6902447       32.2228966       37.7401695       43.3357124       47.2231560       50.2357903       45.4426193       45.3702049       46.6180611       22.6054878       32.2088509       9.33832169       9.33828163       7.31555843       5.06670427      -2.14698148       1.79859817      -1.54654944       100.111786    

 Bootstrap number:     3/    5  Chisq      0.94325
   2.2684705200666047       0.49962374818495314        128.61500820932559        128.57173104453679       0.34510221602215002     
   67.8571854       132.007446       153.807938       163.831650       167.344360       181.251724       188.059418       178.985413       191.464005       188.273941       187.315262       189.743820       186.408218       189.550720       193.104416       194.798416       190.436569       193.218552       192.286636       191.925812       193.857559       193.353348       192.472946       194.471680       194.334244       5.14400959       24.1415501       32.1470299       37.8201103       31.0369644       46.9821434       44.3035278       41.6043625       48.9497566       36.4029503       31.0459366       18.9595642       5.92480326       2.76654625       1.84972370       4.67651987     -0.936659515       2.76113391     -0.774512231       5.96005678       17.3181190       26.9460354       28.6091347       27.9368191       44.0752068       46.2527084       39.1072960       39.3640862       37.4779854       21.7336025       18.8777714       8.04277897       6.35411215       9.30295181       7.85303020      -3.07984185      -1.04339600      0.621575415       99.9820633    

 Bootstrap number:     4/    5  Chisq      0.96506
   2.2673776917649762       0.50382744277370040        128.63366528342473        128.55655682468054       0.25910408444802635     
   73.5412216       137.237671       167.149719       175.668793       186.365784       192.398804       200.815674       196.606567       199.898773       203.618881       196.451782       201.264679       188.448715       189.554871       194.514099       195.354263       187.955902       192.602676       190.568085       192.537979       192.875626       193.345520       192.701279       194.354111       194.343491       11.5318441       28.2508965       41.7924919       45.9835548       48.5931320       59.6343880       58.0652046       56.1668015       56.3521957       49.0352135       36.0495148       28.1552277       9.50198460       2.81840587       5.29429960      0.482863545      -1.82616985       3.54558372      -3.49790096       6.56271362       22.5526485       38.2086906       37.3283119       49.2956924       49.0863304       56.9070244       52.0629387       50.9645233       45.0193443       31.4339142       31.2990685       10.0625992       6.50386810       6.98260593       5.29253674      -2.93947482       1.29398012      -2.38018966       99.9834137    

 Bootstrap number:     5/    5  Chisq      0.92857
   2.2695257395462511       0.50288200156221585        128.59520783237400        128.58204340250799       0.27604047358532302     
   71.3057938       136.506363       164.621735       172.436462       182.568863       182.652664       197.794952       193.226990       194.628891       199.512222       196.255264       197.643188       189.934296       190.779114       194.562057       196.297165       190.666061       193.379089       191.788406       191.320511       192.437958       194.596939       192.929657       194.496094       194.442932       11.9887295       26.6472702       41.4388046       41.1210442       48.7124863       46.6136513       57.7261124       51.3828316       51.6587715       46.5958443       34.7100754       26.4226112       10.6711426       4.75680399       4.85997915       5.05730629      0.825999081       5.12108278      -1.56743908       8.14922810       20.6027107       35.5970268       31.0113392       44.1127701       46.1466942       53.1044807       47.4342728       46.9896889       44.6471596       28.1497669       29.7334690       9.21043396       10.2873907       5.28962421       6.59211063     -0.582140505      -1.19120002       7.31457630E-03   99.9804077    

