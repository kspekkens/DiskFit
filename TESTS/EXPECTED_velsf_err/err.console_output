 DiskFit (v1.2.2)  Copyright (C) 2015, 2017, Jerry Sellwood and Kristine Spekkens

 This program comes with ABSOLUTELY NO WARRANTY.  It is free software
 and you are welcome to redistribute it under certain conditions.
 See <http://www.gnu.org/licenses/> for details.

 Enter input parameter file name

 Read in input parameters from: EXAMPLE/VELS/FITS/velsf_err.inp                                                                     

Input disk eps: (1-b/a)  0.45
Input disk PA (N->E; deg) =  45.00
Input x-center (data units) =   130.50
Input y-center (data units) =   126.50
Input disk systemic velocity (km/s) =   110.00

 Fitting for:
 disk PA
 disk ellipticity
 disk x-center, y-center
 non-axisymmetric disk components
 PA of non-axisymmetric component
 disk systemic velocity
 --> Other attributes fixed to input values

 Velocity field components extracted at radii:
    3.0    9.0   15.0   21.0   27.0   33.0   39.0   45.0   51.0   57.0
   63.0   69.0   75.0   81.0   87.0   93.0   99.0  105.0  111.0  117.0
  123.0  129.0  135.0  141.0  147.0

 Fitting for m = 2 bisymmetric flow (radial and tangential components)
 in addition to rotation velocities
 Non-axisymm flows set to 0 in rings beyond    111.000000    

 Read in measured velocities and uncertainties from:
 EXAMPLE/VELS/FITS/vels.fits


 Number of datapoints read in:        1224
 Number of datapoints to be used in fit:        1224
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
starting iteration    1 best reduced chisq so far: 1.669E+01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
s starting iteration    2 best reduced chisq so far: 2.542E+00
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.2138  0.504227  126.6735  127.9759   76.7718
starting iteration    3 best reduced chisq so far: 1.052E+00
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.0041  0.507169  128.5579  128.5576  110.1634
starting iteration    4 best reduced chisq so far: 1.046E+00
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.0894  0.502896  128.5682  128.5750  108.4181
starting iteration    5 best reduced chisq so far: 1.046E+00
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.0603  0.502659  128.5682  128.5750  108.4166

 Done minimization
 Total number of iterations:            5
 Minimum chi^2 found:    1.04552436    

 Wrote:  EXAMPLE/VELS/FITS/OUT/err.mod.fits
 Wrote:  EXAMPLE/VELS/FITS/OUT/err.res.fits

 Determining uncertainties...
 Beginning bootstrap


 Bootstrap number:            1
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
starting iteration    1 best reduced chisq so far: 1.730E+01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
starting iteration    2 best reduced chisq so far: 2.565E+00
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.3053  0.502973  126.7036  127.9877   84.9306
starting iteration    3 best reduced chisq so far: 9.137E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   39.9845  0.499498  128.5106  128.5978  119.6576
starting iteration    4 best reduced chisq so far: 9.012E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.0016  0.503008  128.5453  128.5972  114.2914
starting iteration    5 best reduced chisq so far: 9.009E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.0117  0.503450  128.5492  128.5975  113.7252
starting iteration    6 best reduced chisq so far: 9.009E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.0142  0.503502  128.5492  128.5975  113.6494

 Done minimization
 Total number of iterations:            6
 Minimum chi^2 found:   0.900910437    



 Bootstrap number:            2
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
starting iteration    1 best reduced chisq so far: 1.722E+01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
starting iteration    2 best reduced chisq so far: 2.545E+00
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.3110  0.497481  126.6422  127.9929   77.0708
starting iteration    3 best reduced chisq so far: 9.365E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   39.9998  0.502675  128.4769  128.5931  110.6776
starting iteration    4 best reduced chisq so far: 9.332E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.0547  0.502641  128.4933  128.5801  106.8378
starting iteration    5 best reduced chisq so far: 9.331E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.0513  0.503135  128.4927  128.5792  106.4044

 Done minimization
 Total number of iterations:            5
 Minimum chi^2 found:   0.933116078    



 Bootstrap number:            3
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
starting iteration    1 best reduced chisq so far: 1.665E+01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
 - - - - - - - - - - - -s starting iteration    2 best reduced chisq so far: 2.528E+00
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.2644  0.499239  126.7954  128.0101   80.3800
starting iteration    3 best reduced chisq so far: 9.952E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   39.9398  0.503740  128.4459  128.5228    4.7646
starting iteration    4 best reduced chisq so far: 9.504E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   39.9456  0.501109  128.5501  128.5580   14.7680
starting iteration    5 best reduced chisq so far: 9.433E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   39.9804  0.499874  128.6159  128.5714   19.6839
starting iteration    6 best reduced chisq so far: 9.433E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   39.9735  0.499882  128.6166  128.5725   19.7373

 Done minimization
 Total number of iterations:            6
 Minimum chi^2 found:   0.943254054    



 Bootstrap number:            4
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
starting iteration    1 best reduced chisq so far: 1.663E+01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
starting iteration    2 best reduced chisq so far: 2.455E+00
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.1953  0.501502  126.8146  127.9861   77.0104
starting iteration    3 best reduced chisq so far: 9.683E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   39.9199  0.501329  128.6233  128.5449  108.4104
starting iteration    4 best reduced chisq so far: 9.651E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   39.9046  0.503572  128.6337  128.5566  104.9053
starting iteration    5 best reduced chisq so far: 9.651E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   39.9086  0.503838  128.6337  128.5566  104.8301

 Done minimization
 Total number of iterations:            5
 Minimum chi^2 found:   0.965064764    



 Bootstrap number:            5
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
starting iteration    1 best reduced chisq so far: 1.676E+01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   45.0000  0.450000  130.5000  126.5000   45.0000
starting iteration    2 best reduced chisq so far: 2.424E+00
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.2145  0.499656  126.7707  127.9985   77.3740
starting iteration    3 best reduced chisq so far: 9.355E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   39.9282  0.501893  128.5695  128.5719  111.0540
starting iteration    4 best reduced chisq so far: 9.288E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.0334  0.502181  128.5947  128.5829  106.3115
starting iteration    5 best reduced chisq so far: 9.286E-01
   diskpa:   diskel:     xcen:     ycen:    phi_b:
   40.0312  0.502863  128.5952  128.5820  105.8350

 Done minimization
 Total number of iterations:            5
 Minimum chi^2 found:   0.928571582    



 Finished           5  bootstrap iterations
 Wrote out: EXAMPLE/VELS/FITS/OUT/err.out
