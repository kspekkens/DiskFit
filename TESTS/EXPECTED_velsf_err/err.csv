disk pa,disk el,xcen,ycen,bar pa
  4.0014E+01,  5.0350E-01,  1.2855E+02,  1.2860E+02,  2.3649E+01
  4.0052E+01,  5.0314E-01,  1.2849E+02,  1.2858E+02,  1.6404E+01
  3.9974E+01,  4.9962E-01,  1.2862E+02,  1.2857E+02,  1.9773E+01
  3.9911E+01,  5.0383E-01,  1.2863E+02,  1.2856E+02,  1.4846E+01
  4.0034E+01,  5.0288E-01,  1.2860E+02,  1.2858E+02,  1.5816E+01
