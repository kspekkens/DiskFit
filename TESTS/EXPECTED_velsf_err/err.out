Minimization output, vels
-------------------------
Input files:
EXAMPLE/VELS/FITS/velsf_err.inp
EXAMPLE/VELS/FITS/vels.fits   pixscale  1.00 arcsec/pix


FITS region:       x-low:      1   y-low:      1 x-range:    255 y-range:    255
FITS sampling:    regrad:  12.00   regpa:  45.00  regeps:   0.45 istpout:      6

Output model and (data-model) residuals files:
EXAMPLE/VELS/FITS/OUT/err.mod
EXAMPLE/VELS/FITS/OUT/err.res

Disk toggles:         PA: T     eps: T  center: T non-axi: T    phib: T
vels toggles:    interp0: T  radial: F    Vsys: T    warp: F      rw: F    welm: F   wphim: F

Input values
------------
disk PA, phi_d^prime (deg):           45.00
disk eps:                              0.45
x,y center (data units):             130.50  126.50
Non-axisymm phib (deg):               45.00
Harmonic order m:                      2
Vsys (km/s):                         110.00
Delta_ISM (km/s):                      2.80





No seeing correction applied
No model component smoothing applied
Uncertainties estimated via bootstrap:  seed:      -50  nunc:        5  junc:   -1.00
Non-axisymmetries fixed to 0 outside range:    3.00  111.00


Best fitting values
-------------------
disk PA, phi_d^prime (deg):           40.06 +/-  0.05
disk eps:                              0.50 +/-  0.00
disk incl (deg):                      60.18 +/-  0.03
x,y center (data units):             128.57 +/-  0.05,  128.58 +/-  0.01
Non-axisymm phib (disk plane, deg):   108.42 +/-  3.02,  -16.14,   49.46
Vsys (km/s):                         100.01 +/-  0.00









Minimization Details
---------------------
# points Dn used in fit:               1224
# iterations in minimization:             5
Minimum chi^2 found:               1.045524
Degrees of freedom in fit:             1155

Fitted velocity components                  (radii in data units, velocities in km/s):
       r        npts          Vt         eVt          Vr         eVr        Vm,t       eVm,t        Vm,r       eVm,r
--------------------------------------------------------------------------------------------------------------------
      3.00     56.28       69.65        2.09        0.00        0.00       -8.61        2.64       -7.41        1.17
      9.00    134.86      132.91        2.00        0.00        0.00      -24.34        2.85      -18.76        3.73
     15.00     60.08      158.38        5.92        0.00        0.00      -35.86        4.00      -31.23        4.65
     21.00     11.62      169.94        5.53        0.00        0.00      -42.12        4.04      -32.00        5.17
     27.00     13.79      171.73        9.57        0.00        0.00      -33.27        8.45      -37.07        9.95
     33.00     16.40      181.61        8.46        0.00        0.00      -48.01        7.95      -42.33        2.20
     39.00     21.65      188.52        9.38        0.00        0.00      -46.83        8.08      -47.27        6.64
     45.00     22.87      183.72        9.58        0.00        0.00      -43.29        9.07      -40.88        5.88
     51.00     26.15      189.78        3.62        0.00        0.00      -47.38        2.74      -41.90        7.35
     57.00     29.91      192.03        9.45        0.00        0.00      -43.03        7.00      -38.43        3.26
     63.00     33.62      190.00        5.86        0.00        0.00      -31.71        2.55      -24.71        5.04
     69.00     34.91      192.19        6.91        0.00        0.00      -20.21        3.18      -22.43        6.63
     75.00     38.79      189.01        0.07        0.00        0.00       -9.75        1.75      -10.58        1.19
     81.00     43.59      189.80        1.57        0.00        0.00       -2.93        1.89       -7.75        2.05
     87.00     44.63      194.18        0.76        0.00        0.00       -4.37        2.32       -6.98        0.50
     93.00     47.45      194.32        0.55        0.00        0.00       -3.53        0.50       -5.74        2.00
     99.00     53.03      190.11        0.31        0.00        0.00        0.14        1.68        1.16        1.49
    105.00     54.28      193.58        0.79        0.00        0.00       -3.23        1.22       -0.20        1.49
    111.00     57.61      192.64        0.92        0.00        0.00       -0.21        1.30       -0.77        1.20
    117.00     60.19      192.47        0.74        0.00        0.00        0.00        0.00        0.00        0.00
    123.00     65.76      192.75        0.79        0.00        0.00        0.00        0.00        0.00        0.00
    129.00     66.17      193.45        0.57        0.00        0.00        0.00        0.00        0.00        0.00
    135.00     69.60      193.09        0.18        0.00        0.00        0.00        0.00        0.00        0.00
    141.00     74.52      194.53        0.02        0.00        0.00        0.00        0.00        0.00        0.00
    147.00     81.61      194.58        0.06        0.00        0.00        0.00        0.00        0.00        0.00
